src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/reissuefurtherevidence/ReissueFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.reissuefurtherevidence;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.DocumentType.*;
+import static uk.gov.hmcts.reform.sscs.ccd.presubmit.actionfurtherevidence.OriginalSenderItemList.APPELLANT;
+import static uk.gov.hmcts.reform.sscs.ccd.presubmit.actionfurtherevidence.OriginalSenderItemList.DWP;
+import static uk.gov.hmcts.reform.sscs.ccd.presubmit.actionfurtherevidence.OriginalSenderItemList.REPRESENTATIVE;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Service
+public class ReissueFurtherEvidenceAboutToSubmitHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    private static Map<String, String> originalSenderToDocumentType = new HashMap<>();
+
+    static {
+        originalSenderToDocumentType.put(APPELLANT.getCode(), APPELLANT_EVIDENCE.getValue());
+        originalSenderToDocumentType.put(REPRESENTATIVE.getCode(), REPRESENTATIVE_EVIDENCE.getValue());
```

Comment:
Shouldn't it only populate this option if the case has a rep on the case?

Reply:
This is a static lookup, not based on any single appeal. Used to check what to set the document type to based on what the original sender has been selected in the mid-Event. (The about to start handler - clears out the original sender - this is because we're not going to be using the midEvent that allows the user to pick the new original sender on this ticket)